CMakeLists.txt

```diff
@@ -28,6 +28,12 @@ if(Git_FOUND)
     endif()
 endif() #git
 
+include (ProcessorCount)
+if (NOT CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  include (CodeCoverage)
+  include (RippledCov)
```

Comment:
Why aren't these includes conditional on the `coverage` option?

Reply:
thanks, I forgot to clean up this part.